java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -127,6 +606,14 @@
     private static final AtomicReference<Map<String, MantaErrorCode>> LOOKUP =
```

Comment:
I suggest that you change from lazy instantiation here to instantiation explicitly within the static scope.

Reply:
Done, Kindly Review the new changes made.